modules/tide_webform/config/optional/jsonapi_extras.jsonapi_resource_config.webform--webform.yml

```diff
@@ -125,7 +125,7 @@ resourceFields:
     fieldName: elements
     publicName: elements
     enhancer:
-      id: default_maxlength_enhancer
```

Comment:
this enhancer is not in use anymore and I have checked all the project. But still kept the code so if we need this in future.